schemas/stsci.edu/asdf/transform/sersic1d-1.0.0.yaml

```diff
@@ -0,0 +1,33 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/transform/sersic1d-1.0.0"
+tag: "tag:stsci.edu:asdf/transform/sersic1d-1.0.0"
+title: >
+  One dimensional Sersic surface brightness profile.
+
+description: >
+  One dimensional Sersic surface brightness profile.
+
+allOf:
+  - $ref: "transform-1.2.0"
+  - type: object
+    properties:
+      amplitude:
+        anyOf:
+          - $ref: "../unit/quantity-1.1.0"
+          - type: number
+        description: Surface brightness at r_eff.
+      r_eff:
+        anyOf:
+          - $ref: "../unit/quantity-1.1.0"
+          - type: number
+        description: Effective (half-light) radius.
+      n:
```

Comment:
@nden @perrygreenfield do you have an opinion on matching the modeling package parameter names vs using a name that is more descriptive?  If someone were to inspect an asdf file written under this schema, they'd just see `n` and the schema description wouldn't be readily available.  We could name it `sersic_index`, for example.

Reply:
I think it would be good practice for the ASDF representation to be more descriptive. Nothing requires it to use the same name that the modeling package uses.

Reply:
in this particular case i personally think the parameter n is widely understood to be the sersic index, but there are certainly other cases where this issue would apply

Reply:
I think names which are widely accepted should be used both in modeling and asdf.
For models which have definitions in the VO standard I think we should try to use the same names. I was going to suggest checking all names against the VO definitions when moving the transforms out of the standard.